packages/coinstac-api-server/src/data/subscriptions/index.js

```diff
@@ -27,6 +27,7 @@ const initSubscriptions = (pubsub) => {
     helperFunctions.getRethinkConnection()
       .then(connection => rethink.table(table.tableName).changes().run(connection))
       .then(feed => feed.each((err, change) => {
+        // eslint-disable-next-line
```

Comment:
in general, disables should disable a specific set of rules, otherwise the line is blind to lint